book/python/using-connectors.md

```diff
@@ -0,0 +1,193 @@
+# Using Connectors As Sources & Sinks
+
+Wallaroo's connectors are an experimental feature that allows full customization of sources and sinks. These streams can be integrated into Wallaroo using libraries you're already familiar with in Python. A number of built-in connectors offer a quick way to get started and hook up common stream types and they're all available for easy customization.
```

Comment:
can we change experimental to language that is more along the lines of "a preview release"